cypress/e2e/specialFeatures/atiAnalytics/index.cy.js

```diff
@@ -129,6 +129,42 @@ const canonicalTestSuites = [
     useReverb: true,
     tests: [assertPageView],
   },
+  {
+    path: '/dari',
+    runforEnv: ['local'],
+    service: 'dari',
+    pageIdentifier: 'dari.page',
```

Comment:
We need to add `siteId` to all dari entries in this list
```suggestion
    pageIdentifier: 'dari.page',
    siteId: 142,
```